sqlite3_vendor/sqlite3_cmakelists.txt

```diff
@@ -8,4 +9,4 @@ install(TARGETS sqlite3
         LIBRARY DESTINATION lib
         ARCHIVE DESTINATION lib
         RUNTIME DESTINATION bin)
-install(FILES sqlite3.h sqlite3ext.h DESTINATION include)
\ No newline at end of file
+install(FILES sqlite3.h sqlite3ext.h DESTINATION include)
```

Comment:
I can't quite see it, but what's the diff here?

Reply:
A newline was added at the end of the file (my editor does this automatically). Let me know if this needs to be changed back

Reply:
it's all good. I was just wondering as GitHub doesn't really seem to render this change.